src/Stratis.Bitcoin.Features.Consensus/CoinViews/Coindb/LeveldbCoindb.cs

```diff
@@ -72,6 +72,59 @@ public void Initialize()
             var options = new Options { CreateIfMissing = true };
             this.leveldb = new DB(options, this.dataFolder);
 
+            // Check if key bytes are in the wrong endian order.
+            HashHeightPair current = this.GetTipHash();
```

Comment:
@quantumagi just again, how would this not happen every time we rebuild the table? Is this suppose to be reversing the table index?

Reply:
Ah nevermind, I understand it 👍